components/brave_new_tab_ui/components/default/vpn/vpn_card.tsx

```diff
@@ -215,10 +226,17 @@ export const VPNPromoWidget = () => {
         </PoweredBy>
         <FeatureList />
         <ActionArea>
-          <ActionButton onClick={() => dispatch(Actions.openVPNAccountPage())}>
+          <ActionButton
+            onClick={() => dispatch(Actions.openVPNAccountPage('checkout'))}
+          >
             {getLocale('braveVpnCTA')}
           </ActionButton>
-          <ActionLabel>{getLocale('braveVpnFreeTrial')}</ActionLabel>
```

Comment:
did we intentionally remove the free trial?

Reply:
Yeah, that label is replaced with recover link button.